Automattic/wp-calypso
# Onboarding standalone installation: Display Jetpack specific plugin images

#### Proposed Changes

* Display specific Jetpack plugin images for Backup, Social, Search & Boost product purchases. The rest of the product will still display the old Jetpack plugin image.
* Display a new footer background for standalone installation instructions.
* Move contact us text to the left for standalone installation instructions.

<img width="1223" alt="Screen Shot 2022-10-27 at 2 56 54 PM" src="https://user-images.githubusercontent.com/56598660/198213100-7090af78-46d5-4a0b-832b-fb6d1bf24510.png">


#### Testing Instructions

1. First we need to generate a **receipt ID** so we can properly verify the new screen.
    * Go to https://cloud.jetpack.com/pricing
    * Purchase a **Jetpack Social** 
    * Proceed to checkout and complete the payment
    * Once you land in the thank you page, copy the receipt ID in the URL. 
    
    <img width="1723" alt="Screen Shot 2022-10-24 at 3 46 40 PM" src="https://user-images.githubusercontent.com/56598660/197482629-70f4c1ff-aae5-4a7a-bd19-324e08948fa0.png">
    
2. Now run this branch by following the steps below (or you can use the Calypso live link commented on this PR)
    * Run `git fetch && git checkout add/jetpack-standalone-plugins-onboarding-image`
    * Run `yarn start`

4. Go to http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-manual-activate-instructions/PRODUCT_SLUG?receiptId=RECEIPT_ID or by using the Calypso live link below and appending `/checkout/jetpack/thank-you/licensing-manual-activate-instructions/PRODUCT_SLUG?receiptId=RECEIPT_ID&flags=jetpack/standalone-plugin-onboarding-update-v1`
   * make sure that you replace the PRODUCT_SLUG with a supported product slug (refer to the table below)
   * Make sure that you replace RECEIPT_ID with a valid receipt ID. Use the one you copied earlier.
   * Confirm that **Jetpack Social, Search, Backup and Boost** _displays the correct image_, _contact us text is on the left_ and _new background image on the right_. refer to the table below for all the product slug available.
   
<img width="393" alt="Screen Shot 2022-10-27 at 2 38 38 PM" src="https://user-images.githubusercontent.com/56598660/198209912-9d6bc447-937e-452e-98c2-22feaa37c8ad.png"> <img width="386" alt="Screen Shot 2022-10-27 at 2 40 22 PM" src="https://user-images.githubusercontent.com/56598660/198209920-ef25cb21-9f05-443d-a0f8-cc22d8878865.png"> <img width="389" alt="Screen Shot 2022-10-27 at 2 39 00 PM" src="https://user-images.githubusercontent.com/56598660/198209923-52efd5ba-7fee-4c22-b4e2-5188cacdcc81.png"> <img width="388" alt="Screen Shot 2022-10-27 at 2 39 55 PM" src="https://user-images.githubusercontent.com/56598660/198209927-679bd8d9-f86f-4351-a90a-058c0360c43a.png"> 
<img width="394" alt="Screen Shot 2022-10-31 at 1 44 46 PM" src="https://user-images.githubusercontent.com/56598660/198939778-6b40e904-bc45-486f-a336-db0933927a2e.png">

   
   * Confirm the rest of the product slugs shows the old image and background.

   
<img width="1223" alt="Screen Shot 2022-10-27 at 2 37 31 PM" src="https://user-images.githubusercontent.com/56598660/198209330-b13e6431-1271-43be-a5ea-6fa60879cd09.png">

### Reference table

#### Pre-merge Checklist

| Product Slug        | With new images  |
| ------------- |:-------------:| 
|  jetpack_boost_yearly | ✅  |
| jetpack_boost_monthly | ✅  |
| jetpack_backup | ✅  |
| jetpack_backup_t1_yearly | ✅  |
| jetpack_backup_t1_monthly | ✅  |
| jetpack_backup_t2_yearly | ✅  |
| jetpack_backup_t2_monthly | ✅  |
| jetpack_backup_daily | ✅  |
| jetpack_backup_realtime | ✅  |
| jetpack_backup_daily_monthly | ✅  |
| jetpack_backup_realtime_monthly | ✅  |
| jetpack_scan | ❌  |
| jetpack_scan_monthly | ❌  |
| jetpack_scan_realtime | ❌  |
| jetpack_scan_realtime_monthly | ❌  |
| jetpack_anti_spam | ❌  |
| jetpack_anti_spam_monthly | ❌  |
| jetpack_search | ✅  |
| jetpack_search_free | ✅  |
| jetpack_search_monthly | ✅  |
| jetpack_crm | ✅  |
| jetpack_crm_monthly | ✅  |
| jetpack_videopress | ✅   |
| jetpack_videopress_monthly | ✅   |
| jetpack_social_basic_yearly | ✅ |
| jetpack_social_basic_monthly | ✅ |
| jetpack_personal | ❌  |
| jetpack_personal_monthly | ❌  |
| jetpack_premium | ❌  |
| jetpack_premium_monthly | ❌  |
| jetpack_business | ❌  |
| jetpack_business_monthly | ❌  |
| jetpack_security_t1_yearly | ❌  |
| jetpack_security_t1_monthly | ❌  |
| jetpack_security_t2_yearly | ❌  |
| jetpack_security_t2_monthly | ❌  |
| jetpack_complete | ❌  |
| jetpack_complete_monthly | ❌  |
| jetpack_security_daily | ❌  |
| jetpack_security_daily_monthly | ❌  |
| jetpack_security_realtime | ❌  |
| jetpack_security_realtime_monthly | ❌  |






Related to 1203097885147382-as-1203159154260045
